lib/licensee/projects/github_project.rb

```diff
@@ -41,8 +43,14 @@ def files
       end
 
       def load_file(file)
-        client.contents(@repo, path:   file[:path],
-                               accept: 'application/vnd.github.v3.raw').to_s
+        pathname = file[:path]
+        accept = 'application/vnd.github.v3.raw'
+        ref = @revision
+        if @revision
+          client.contents(@repo, path: pathname, accept: accept, ref: ref).to_s
+        else
+          client.contents(@repo, path: pathname, accept: accept).to_s
```

Comment:
Both of these look the same, I _think_ if `@revision` is nil, and is passed as a `ref` argument, the behavior should be the same?

Reply:
You would think that, but it was adding the param to the URL without a value and causing problems! :/ I had it as your described previously.